src/cc_catalog_airflow/dags/provider_api_scripts/storage/constants.py

```diff
@@ -0,0 +1,65 @@
+_SIMPLE_LICENSE_PATHS = [
+    # This list holds valid license URL path snippets that split
+    # correctly into license_ and license_version
+    'by/1.0',
+    'by/2.0',
+    'by/2.5',
+    'by/3.0',
+    'by/4.0',
+
+    'by-nc/1.0',
+    'by-nc/2.0',
+    'by-nc/2.5',
+    'by-nc/3.0',
+    'by-nc/4.0',
+
+    'by-nc-nd/1.0',
+    'by-nc-nd/2.0',
+    'by-nc-nd/2.5',
+    'by-nc-nd/3.0',
+    'by-nc-nd/4.0',
+
+    'by-nc-sa/1.0',
+    'by-nc-sa/2.0',
+    'by-nc-sa/2.5',
+    'by-nc-sa/3.0',
+    'by-nc-sa/4.0',
+
+    'by-nd/1.0',
+    'by-nd/2.0',
+    'by-nd/2.5',
+    'by-nd/3.0',
+    'by-nd/4.0',
+
+    'by-sa/1.0',
+    'by-sa/2.0',
+    'by-sa/2.5',
+    'by-sa/3.0',
+    'by-sa/4.0',
+
+    'cc0/1.0',
+
+    'sa/1.0',
+]
```

Comment:
Would it be beneficial toward that end to specify `/licenses/`?  The only way I can image it would help would be if there were different licenses that included one of the above snippets, but were distinguished by the `/licenses/` portion of the path.

For the country distinction, I'll talk it over with @aldenstpage .